package.json

```diff
@@ -8,9 +8,12 @@
   "repository": "https://github.com/thysultan/stylis.js",
   "bugs": "https://github.com/thysultan/stylis.js/issues",
   "type": "module",
-  "main": "dist/stylis.umd.js",
+  "main": "dist/stylis.cjs",
```

Comment:
This should be 'umd.js' unless i've missed something?

Reply:
I've merged the other PR after seeing [the LGTM comment](https://github.com/thysultan/stylis.js/pull/207#issuecomment-614642816). Changed this to UMD just now: https://github.com/thysultan/stylis.js/pull/195/commits/395454b8e75c627db52dc5cd72b91e169c18d774